src/alert/lsst.rs

```diff
@@ -885,6 +897,7 @@ impl AlertWorker for LsstAlertWorker {
                     &prv_candidates_doc,
                     &prv_nondetections_doc,
                     &fp_hist_doc,
+                    &Document::new(),
```

Comment:
might just want to make it an option so you don't need to create a new document when you wouldn't use it anyways?

Reply:
Yeah that occurred to me, I wanted to see how you reacted first. I wasn't sure if crossmatching between surveys will be normal/common enough that we'd always be providing `Some(doc)` in practice. If survey crossmatching truly is going to be an optional or extra thing, then my preference to make it `Option<Document>`.

Reply:
And this is what I struggle with about nosql, everything is optional all of the time forcing me to think and make decisions :p

Reply:
yeah I know hahaha

So really the situation is that the priority is ZTF being crossmatched with LSST, but long term we're hoping to do the opposite, if the performance is good enough that we can crossmatch 10M LSST alerts every night with ZTF, which is unclear with a MongoDB based implementation.

So let's just keep it as is for now.

Reply:
Haha I just committed it to Option. We can change it later when it makes sense.